engine/src/flutter/display_list/utils/dl_matrix_clip_tracker.cc

```diff
@@ -299,6 +330,28 @@ bool DisplayListMatrixClipState::rrect_covers_cull(
   return true;
 }
 
+bool DisplayListMatrixClipState::rsuperellipse_covers_cull(
+    const DlRoundSuperellipse& content) const {
+  if (content.IsEmpty()) {
+    return false;
+  }
+  if (cull_rect_.IsEmpty()) {
+    return true;
+  }
+  if (content.IsRect()) {
+    return rect_covers_cull(content.GetBounds());
+  }
+  if (content.IsOval()) {
+    return oval_covers_cull(content.GetBounds());
+  }
+  DlPoint corners[4];
+  if (!getLocalCullCorners(corners)) {
+    return false;
+  }
+  // TODO(dkwingsmt): Implement Contains
+  return true;
```

Comment:
Return false as we haven't proven the "cover"ing state.

Reply:
I've implemented `Contains` here.